[2025-05-23T02:13:41.735+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-23T02:13:41.745+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DYNAMIC_FILE_TO_DB_LOADER.transfer_files scheduled__2025-05-23T01:20:00+00:00 map_index=1 [queued]>
[2025-05-23T02:13:41.755+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DYNAMIC_FILE_TO_DB_LOADER.transfer_files scheduled__2025-05-23T01:20:00+00:00 map_index=1 [queued]>
[2025-05-23T02:13:41.755+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-05-23T02:13:41.763+0000] {taskinstance.py:2890} INFO - Executing <Mapped(KubernetesPodOperator): transfer_files> on 2025-05-23 01:20:00+00:00
[2025-05-23T02:13:41.768+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=16323) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-23T02:13:41.769+0000] {standard_task_runner.py:72} INFO - Started process 16326 to run task
[2025-05-23T02:13:41.768+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'DYNAMIC_FILE_TO_DB_LOADER', 'transfer_files', 'scheduled__2025-05-23T01:20:00+00:00', '--job-id', '1975', '--raw', '--subdir', 'DAGS_FOLDER/***_dynamic_file_to_db_loader.py', '--cfg-path', '/tmp/tmpgwm2dhgw', '--map-index', '1']
[2025-05-23T02:13:41.770+0000] {standard_task_runner.py:105} INFO - Job 1975: Subtask transfer_files
[2025-05-23T02:13:41.800+0000] {task_command.py:467} INFO - Running <TaskInstance: DYNAMIC_FILE_TO_DB_LOADER.transfer_files scheduled__2025-05-23T01:20:00+00:00 map_index=1 [running]> on host 5fb7fe4c544d
[2025-05-23T02:13:41.846+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='DYNAMIC_FILE_TO_DB_LOADER' AIRFLOW_CTX_TASK_ID='transfer_files' AIRFLOW_CTX_EXECUTION_DATE='2025-05-23T01:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-23T01:20:00+00:00'
[2025-05-23T02:13:41.847+0000] {logging_mixin.py:190} INFO - 
        INSERT INTO ***_daily_log (log_time, source_name, target_name)
        VALUES (CURRENT_TIMESTAMP, %(source)s, %(target)s)
    
[2025-05-23T02:13:41.851+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-05-23T02:13:41.853+0000] {sql.py:511} INFO - Running statement: 
        INSERT INTO ***_daily_log (log_time, source_name, target_name)
        VALUES (CURRENT_TIMESTAMP, %(source)s, %(target)s)
    , parameters: {'source': 'xyz.xlsx', 'target': 'ahajiyev/xyz'}
[2025-05-23T02:13:41.854+0000] {sql.py:520} INFO - Rows affected: 1
[2025-05-23T02:13:41.855+0000] {logging_mixin.py:190} INFO - 
        UPDATE ***_daily_log
        SET dag_id = %(dag_id)s,
            task_id = %(task_id)s,
            run_id = %(run_id)s,
            execution_date = %(execution_date)s,
            try_number = %(try_number)s,
            log_time = CURRENT_TIMESTAMP
        WHERE date(log_time) = date(CURRENT_DATE)
        and dag_id is null
          AND source_name = %(source)s
          AND target_name = %(target)s
    
[2025-05-23T02:13:41.859+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-05-23T02:13:41.862+0000] {sql.py:511} INFO - Running statement: 
        UPDATE ***_daily_log
        SET dag_id = %(dag_id)s,
            task_id = %(task_id)s,
            run_id = %(run_id)s,
            execution_date = %(execution_date)s,
            try_number = %(try_number)s,
            log_time = CURRENT_TIMESTAMP
        WHERE date(log_time) = date(CURRENT_DATE)
        and dag_id is null
          AND source_name = %(source)s
          AND target_name = %(target)s
    , parameters: {'dag_id': 'DYNAMIC_FILE_TO_DB_LOADER', 'task_id': 'transfer_files', 'run_id': 'scheduled__2025-05-23T01:20:00+00:00', 'execution_date': '2025-05-23 01:20:00+00:00', 'try_number': 1, 'source': 'xyz.xlsx', 'target': 'ahajiyev/xyz'}
[2025-05-23T02:13:41.863+0000] {sql.py:520} INFO - Rows affected: 1
[2025-05-23T02:13:41.864+0000] {logging_mixin.py:190} INFO - 
        UPDATE ***_daily_log
        SET status = %(status)s
        WHERE date(log_time) = date(CURRENT_DATE)
        and try_number = %(try_number)s
          AND source_name = %(source)s
          AND target_name = %(target)s
    
[2025-05-23T02:13:41.869+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-05-23T02:13:41.871+0000] {sql.py:511} INFO - Running statement: 
        UPDATE ***_daily_log
        SET status = %(status)s
        WHERE date(log_time) = date(CURRENT_DATE)
        and try_number = %(try_number)s
          AND source_name = %(source)s
          AND target_name = %(target)s
    , parameters: {'status': 'running', 'try_number': 1, 'source': 'xyz.xlsx', 'target': 'ahajiyev/xyz'}
[2025-05-23T02:13:41.872+0000] {sql.py:520} INFO - Rows affected: 4
[2025-05-23T02:13:41.873+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-23T02:13:41.877+0000] {pod.py:1148} INFO - Building pod excel-to-parquet-s58nz42e with labels: {'dag_id': 'DYNAMIC_FILE_TO_DB_LOADER', 'task_id': 'transfer_files', 'run_id': 'scheduled__2025-05-23T0120000000-444fd191e', 'kubernetes_pod_operator': 'True', 'map_index': '1', 'try_number': '1'}
[2025-05-23T02:13:41.947+0000] {pod.py:545} INFO - Found matching pod excel-to-parquet-s58nz42e with labels {'airflow_kpo_in_cluster': 'False', 'airflow_version': '2.10.5', 'dag_id': 'DYNAMIC_FILE_TO_DB_LOADER', 'kubernetes_pod_operator': 'True', 'map_index': '1', 'run_id': 'scheduled__2025-05-23T0120000000-444fd191e', 'task_id': 'transfer_files', 'try_number': '1'}
[2025-05-23T02:13:41.947+0000] {pod.py:546} INFO - `try_number` of task_instance: 1
[2025-05-23T02:13:41.947+0000] {pod.py:547} INFO - `try_number` of pod: 1
[2025-05-23T02:13:41.952+0000] {pod_manager.py:392} WARNING - Pod not yet started: excel-to-parquet-s58nz42e
[2025-05-23T02:13:46.988+0000] {pod_manager.py:392} WARNING - Pod not yet started: excel-to-parquet-s58nz42e
[2025-05-23T02:13:52.032+0000] {pod_manager.py:477} INFO - [base] Added `s3-local` successfully.
[2025-05-23T02:13:52.033+0000] {pod_manager.py:477} INFO - [base] `s3-local/ahajiyev/dynamic_file_to_db_transfer.py` -> `/tmp/dynamic_file_to_db_transfer.py`
[2025-05-23T02:13:52.033+0000] {pod_manager.py:477} INFO - [base] Total: 2.82 KiB, Transferred: 2.82 KiB, Speed: 62.71 KiB/s
[2025-05-23T02:13:52.033+0000] {pod_manager.py:477} INFO - [base] :: loading settings :: url = jar:file:/opt/spark/jars/ivy-2.5.1.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2025-05-23T02:13:52.034+0000] {pod_manager.py:477} INFO - [base] Ivy Default Cache set to: /root/.ivy2/cache
[2025-05-23T02:13:52.034+0000] {pod_manager.py:477} INFO - [base] The jars for the packages stored in: /root/.ivy2/jars
[2025-05-23T02:13:52.034+0000] {pod_manager.py:477} INFO - [base] com.crealytics#spark-excel_2.12 added as a dependency
[2025-05-23T02:13:52.034+0000] {pod_manager.py:477} INFO - [base] :: resolving dependencies :: org.apache.spark#spark-submit-parent-04c8c5f3-b77b-489b-87d5-412c8f167f99;1.0
[2025-05-23T02:13:53.149+0000] {pod_manager.py:477} INFO - [base] 	confs: [default]
[2025-05-23T02:13:53.381+0000] {pod_manager.py:477} INFO - [base] 	found com.crealytics#spark-excel_2.12;0.13.7 in central
[2025-05-23T02:13:55.315+0000] {pod_manager.py:477} INFO - [base] 	found org.apache.poi#poi;4.1.2 in central
[2025-05-23T02:13:55.641+0000] {pod_manager.py:477} INFO - [base] 	found commons-codec#commons-codec;1.13 in central
[2025-05-23T03:24:39.362+0000] {job.py:229} INFO - Heartbeat recovered after 4237.38 seconds
[2025-05-23T03:24:44.633+0000] {pod_manager.py:477} INFO - [base] 	found org.apache.commons#commons-collections4;4.4 in central
[2025-05-23T03:24:45.775+0000] {pod_manager.py:477} INFO - [base] 	found org.apache.commons#commons-math3;3.6.1 in central
[2025-05-23T03:24:45.955+0000] {pod_manager.py:477} INFO - [base] 	found com.zaxxer#SparseBitSet;1.2 in central
[2025-05-23T03:24:46.196+0000] {pod_manager.py:477} INFO - [base] 	found org.apache.poi#poi-ooxml;4.1.2 in central
[2025-05-23T03:24:46.508+0000] {pod_manager.py:477} INFO - [base] 	found org.apache.poi#poi-ooxml-schemas;4.1.2 in central
[2025-05-23T03:24:46.870+0000] {pod_manager.py:477} INFO - [base] 	found org.apache.xmlbeans#xmlbeans;3.1.0 in central
[2025-05-23T03:24:47.056+0000] {pod_manager.py:477} INFO - [base] 	found com.github.virtuald#curvesapi;1.06 in central
[2025-05-23T03:24:48.346+0000] {pod_manager.py:477} INFO - [base] 	found com.norbitltd#spoiwo_2.12;1.8.0 in central
[2025-05-23T03:24:48.554+0000] {pod_manager.py:477} INFO - [base] 	found org.scala-lang.modules#scala-xml_2.12;1.3.0 in central
[2025-05-23T03:24:48.814+0000] {pod_manager.py:477} INFO - [base] 	found com.github.pjfanning#excel-streaming-reader;2.3.6 in central
[2025-05-23T03:24:49.094+0000] {pod_manager.py:477} INFO - [base] 	found com.github.pjfanning#poi-shared-strings;1.0.4 in central
[2025-05-23T03:24:49.319+0000] {pod_manager.py:477} INFO - [base] 	found com.h2database#h2;1.4.200 in central
[2025-05-23T03:24:49.637+0000] {pod_manager.py:477} INFO - [base] 	found org.apache.commons#commons-text;1.8 in central
[2025-05-23T03:24:49.912+0000] {pod_manager.py:477} INFO - [base] 	found org.apache.commons#commons-lang3;3.9 in central
[2025-05-23T03:24:50.759+0000] {pod_manager.py:477} INFO - [base] 	found xml-apis#xml-apis;1.4.01 in central
[2025-05-23T03:24:51.007+0000] {pod_manager.py:477} INFO - [base] 	found org.slf4j#slf4j-api;1.7.30 in central
[2025-05-23T03:24:51.194+0000] {pod_manager.py:477} INFO - [base] 	found org.apache.commons#commons-compress;1.20 in central
[2025-05-23T03:24:52.001+0000] {pod_manager.py:477} INFO - [base] downloading https://repo1.maven.org/maven2/com/crealytics/spark-excel_2.12/0.13.7/spark-excel_2.12-0.13.7.jar ...
[2025-05-23T03:24:52.083+0000] {pod_manager.py:477} INFO - [base] 	[SUCCESSFUL ] com.crealytics#spark-excel_2.12;0.13.7!spark-excel_2.12.jar (894ms)
[2025-05-23T03:24:52.410+0000] {pod_manager.py:477} INFO - [base] downloading https://repo1.maven.org/maven2/org/apache/poi/poi/4.1.2/poi-4.1.2.jar ...
[2025-05-23T03:24:53.425+0000] {pod_manager.py:477} INFO - [base] 	[SUCCESSFUL ] org.apache.poi#poi;4.1.2!poi.jar (402ms)
[2025-05-23T03:24:53.903+0000] {pod_manager.py:477} INFO - [base] downloading https://repo1.maven.org/maven2/org/apache/poi/poi-ooxml/4.1.2/poi-ooxml-4.1.2.jar ...
[2025-05-23T03:24:53.979+0000] {pod_manager.py:477} INFO - [base] 	[SUCCESSFUL ] org.apache.poi#poi-ooxml;4.1.2!poi-ooxml.jar (1491ms)
[2025-05-23T03:24:54.100+0000] {pod_manager.py:477} INFO - [base] downloading https://repo1.maven.org/maven2/com/norbitltd/spoiwo_2.12/1.8.0/spoiwo_2.12-1.8.0.jar ...
[2025-05-23T03:24:54.170+0000] {pod_manager.py:477} INFO - [base] 	[SUCCESSFUL ] com.norbitltd#spoiwo_2.12;1.8.0!spoiwo_2.12.jar (193ms)
[2025-05-23T03:24:54.248+0000] {pod_manager.py:477} INFO - [base] downloading https://repo1.maven.org/maven2/com/github/pjfanning/excel-streaming-reader/2.3.6/excel-streaming-reader-2.3.6.jar ...
[2025-05-23T03:24:54.319+0000] {pod_manager.py:477} INFO - [base] 	[SUCCESSFUL ] com.github.pjfanning#excel-streaming-reader;2.3.6!excel-streaming-reader.jar (148ms)
[2025-05-23T03:24:54.392+0000] {pod_manager.py:477} INFO - [base] downloading https://repo1.maven.org/maven2/com/github/pjfanning/poi-shared-strings/1.0.4/poi-shared-strings-1.0.4.jar ...
[2025-05-23T03:24:54.465+0000] {pod_manager.py:477} INFO - [base] 	[SUCCESSFUL ] com.github.pjfanning#poi-shared-strings;1.0.4!poi-shared-strings.jar (143ms)
[2025-05-23T03:24:54.586+0000] {pod_manager.py:477} INFO - [base] downloading https://repo1.maven.org/maven2/org/apache/commons/commons-compress/1.20/commons-compress-1.20.jar ...
[2025-05-23T03:24:54.661+0000] {pod_manager.py:477} INFO - [base] 	[SUCCESSFUL ] org.apache.commons#commons-compress;1.20!commons-compress.jar (192ms)
[2025-05-23T03:24:54.768+0000] {pod_manager.py:477} INFO - [base] downloading https://repo1.maven.org/maven2/commons-codec/commons-codec/1.13/commons-codec-1.13.jar ...
[2025-05-23T03:24:54.843+0000] {pod_manager.py:477} INFO - [base] 	[SUCCESSFUL ] commons-codec#commons-codec;1.13!commons-codec.jar (182ms)
[2025-05-23T03:24:54.984+0000] {pod_manager.py:477} INFO - [base] downloading https://repo1.maven.org/maven2/org/apache/commons/commons-collections4/4.4/commons-collections4-4.4.jar ...
[2025-05-23T03:24:55.054+0000] {pod_manager.py:477} INFO - [base] 	[SUCCESSFUL ] org.apache.commons#commons-collections4;4.4!commons-collections4.jar (206ms)
[2025-05-23T03:24:55.300+0000] {pod_manager.py:477} INFO - [base] downloading https://repo1.maven.org/maven2/org/apache/commons/commons-math3/3.6.1/commons-math3-3.6.1.jar ...
[2025-05-23T03:24:55.369+0000] {pod_manager.py:477} INFO - [base] 	[SUCCESSFUL ] org.apache.commons#commons-math3;3.6.1!commons-math3.jar (319ms)
[2025-05-23T03:24:55.446+0000] {pod_manager.py:477} INFO - [base] downloading https://repo1.maven.org/maven2/com/zaxxer/SparseBitSet/1.2/SparseBitSet-1.2.jar ...
[2025-05-23T03:24:55.527+0000] {pod_manager.py:477} INFO - [base] 	[SUCCESSFUL ] com.zaxxer#SparseBitSet;1.2!SparseBitSet.jar (145ms)
[2025-05-23T03:24:56.254+0000] {pod_manager.py:477} INFO - [base] downloading https://repo1.maven.org/maven2/org/apache/poi/poi-ooxml-schemas/4.1.2/poi-ooxml-schemas-4.1.2.jar ...
[2025-05-23T03:24:56.343+0000] {pod_manager.py:477} INFO - [base] 	[SUCCESSFUL ] org.apache.poi#poi-ooxml-schemas;4.1.2!poi-ooxml-schemas.jar (802ms)
[2025-05-23T03:24:56.423+0000] {pod_manager.py:477} INFO - [base] downloading https://repo1.maven.org/maven2/com/github/virtuald/curvesapi/1.06/curvesapi-1.06.jar ...
[2025-05-23T03:24:56.497+0000] {pod_manager.py:477} INFO - [base] 	[SUCCESSFUL ] com.github.virtuald#curvesapi;1.06!curvesapi.jar (164ms)
[2025-05-23T03:24:56.785+0000] {pod_manager.py:477} INFO - [base] downloading https://repo1.maven.org/maven2/org/apache/xmlbeans/xmlbeans/3.1.0/xmlbeans-3.1.0.jar ...
[2025-05-23T03:24:56.861+0000] {pod_manager.py:477} INFO - [base] 	[SUCCESSFUL ] org.apache.xmlbeans#xmlbeans;3.1.0!xmlbeans.jar (359ms)
[2025-05-23T03:24:56.982+0000] {pod_manager.py:477} INFO - [base] downloading https://repo1.maven.org/maven2/org/scala-lang/modules/scala-xml_2.12/1.3.0/scala-xml_2.12-1.3.0.jar ...
[2025-05-23T03:24:57.054+0000] {pod_manager.py:477} INFO - [base] 	[SUCCESSFUL ] org.scala-lang.modules#scala-xml_2.12;1.3.0!scala-xml_2.12.jar(bundle) (195ms)
[2025-05-23T03:24:57.145+0000] {pod_manager.py:477} INFO - [base] downloading https://repo1.maven.org/maven2/xml-apis/xml-apis/1.4.01/xml-apis-1.4.01.jar ...
[2025-05-23T03:24:57.218+0000] {pod_manager.py:477} INFO - [base] 	[SUCCESSFUL ] xml-apis#xml-apis;1.4.01!xml-apis.jar (160ms)
[2025-05-23T03:24:57.295+0000] {pod_manager.py:477} INFO - [base] downloading https://repo1.maven.org/maven2/org/slf4j/slf4j-api/1.7.30/slf4j-api-1.7.30.jar ...
[2025-05-23T03:24:57.372+0000] {pod_manager.py:477} INFO - [base] 	[SUCCESSFUL ] org.slf4j#slf4j-api;1.7.30!slf4j-api.jar (148ms)
[2025-05-23T03:24:57.635+0000] {pod_manager.py:477} INFO - [base] downloading https://repo1.maven.org/maven2/com/h2database/h2/1.4.200/h2-1.4.200.jar ...
[2025-05-23T03:24:57.708+0000] {pod_manager.py:477} INFO - [base] 	[SUCCESSFUL ] com.h2database#h2;1.4.200!h2.jar (337ms)
[2025-05-23T04:04:37.381+0000] {job.py:229} INFO - Heartbeat recovered after 2372.72 seconds
[2025-05-23T04:04:42.414+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2025-05-23T04:04:42.415+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-05-23T04:04:42.416+0000] {process_utils.py:132} INFO - Sending 15 to group 16326. PIDs of all processes in the group: [16326]
[2025-05-23T04:04:42.416+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 16326
[2025-05-23T04:04:42.417+0000] {taskinstance.py:3094} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-05-23T04:04:42.424+0000] {taskinstance.py:3095} ERROR - Stacktrace: 
  File "/home/***/.local/bin/***", line 8, in <module>
    sys.exit(main())
  File "/home/***/.local/lib/python3.12/site-packages/***/__main__.py", line 62, in main
    args.func(args)
  File "/home/***/.local/lib/python3.12/site-packages/***/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/***/.local/lib/python3.12/site-packages/***/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
  File "/home/***/.local/lib/python3.12/site-packages/***/utils/providers_configuration_loader.py", line 55, in wrapped_function
    return func(*args, **kwargs)
  File "/home/***/.local/lib/python3.12/site-packages/***/cli/commands/scheduler_command.py", line 56, in scheduler
    run_command_with_daemon_option(
  File "/home/***/.local/lib/python3.12/site-packages/***/cli/commands/daemon_utils.py", line 86, in run_command_with_daemon_option
    callback()
  File "/home/***/.local/lib/python3.12/site-packages/***/cli/commands/scheduler_command.py", line 59, in <lambda>
    callback=lambda: _run_scheduler_job(args),
  File "/home/***/.local/lib/python3.12/site-packages/***/cli/commands/scheduler_command.py", line 47, in _run_scheduler_job
    run_job(job=job_runner.job, execute_callable=job_runner._execute)
  File "/home/***/.local/lib/python3.12/site-packages/***/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/***/.local/lib/python3.12/site-packages/***/jobs/job.py", line 421, in run_job
    return execute_job(job, execute_callable=execute_callable)
  File "/home/***/.local/lib/python3.12/site-packages/***/jobs/job.py", line 450, in execute_job
    ret = execute_callable()
  File "/home/***/.local/lib/python3.12/site-packages/***/jobs/scheduler_job_runner.py", line 990, in _execute
    executor.start()
  File "/home/***/.local/lib/python3.12/site-packages/***/executors/local_executor.py", line 392, in start
    self.impl.start()
  File "/home/***/.local/lib/python3.12/site-packages/***/executors/local_executor.py", line 330, in start
    worker.start()
  File "/usr/local/lib/python3.12/multiprocessing/process.py", line 121, in start
    self._popen = self._Popen(self)
  File "/usr/local/lib/python3.12/multiprocessing/context.py", line 224, in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
  File "/usr/local/lib/python3.12/multiprocessing/context.py", line 282, in _Popen
    return Popen(process_obj)
  File "/usr/local/lib/python3.12/multiprocessing/popen_fork.py", line 19, in __init__
    self._launch(process_obj)
  File "/usr/local/lib/python3.12/multiprocessing/popen_fork.py", line 71, in _launch
    code = process_obj._bootstrap(parent_sentinel=child_r)
  File "/usr/local/lib/python3.12/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/home/***/.local/lib/python3.12/site-packages/***/executors/local_executor.py", line 80, in run
    return super().run()
  File "/usr/local/lib/python3.12/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/***/.local/lib/python3.12/site-packages/***/traces/tracer.py", line 58, in wrapper
    return func(*args, **kwargs)
  File "/home/***/.local/lib/python3.12/site-packages/***/executors/local_executor.py", line 211, in do_work
    self.execute_work(key=key, command=command)
  File "/home/***/.local/lib/python3.12/site-packages/***/traces/tracer.py", line 58, in wrapper
    return func(*args, **kwargs)
  File "/home/***/.local/lib/python3.12/site-packages/***/executors/local_executor.py", line 100, in execute_work
    state = self._execute_work_in_fork(command)
  File "/home/***/.local/lib/python3.12/site-packages/***/traces/tracer.py", line 58, in wrapper
    return func(*args, **kwargs)
  File "/home/***/.local/lib/python3.12/site-packages/***/executors/local_executor.py", line 142, in _execute_work_in_fork
    args.func(args)
  File "/home/***/.local/lib/python3.12/site-packages/***/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/***/.local/lib/python3.12/site-packages/***/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
  File "/home/***/.local/lib/python3.12/site-packages/***/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/home/***/.local/lib/python3.12/site-packages/***/cli/commands/task_command.py", line 254, in _run_task_by_selected_method
    return _run_task_by_local_task_job(args, ti)
  File "/home/***/.local/lib/python3.12/site-packages/***/cli/commands/task_command.py", line 322, in _run_task_by_local_task_job
    ret = run_job(job=job_runner.job, execute_callable=job_runner._execute)
  File "/home/***/.local/lib/python3.12/site-packages/***/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/***/.local/lib/python3.12/site-packages/***/jobs/job.py", line 421, in run_job
    return execute_job(job, execute_callable=execute_callable)
  File "/home/***/.local/lib/python3.12/site-packages/***/jobs/job.py", line 450, in execute_job
    ret = execute_callable()
  File "/home/***/.local/lib/python3.12/site-packages/***/jobs/local_task_job_runner.py", line 171, in _execute
    self.task_runner.start()
  File "/home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py", line 55, in start
    self.process = self._start_by_fork()
  File "/home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/home/***/.local/lib/python3.12/site-packages/***/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/***/.local/lib/python3.12/site-packages/***/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
  File "/home/***/.local/lib/python3.12/site-packages/***/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/home/***/.local/lib/python3.12/site-packages/***/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/home/***/.local/lib/python3.12/site-packages/***/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/home/***/.local/lib/python3.12/site-packages/***/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/***/.local/lib/python3.12/site-packages/***/models/taskinstance.py", line 3006, in _run_raw_task
    return _run_raw_task(
  File "/home/***/.local/lib/python3.12/site-packages/***/models/taskinstance.py", line 274, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/***/.local/lib/python3.12/site-packages/***/models/taskinstance.py", line 3161, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/***/.local/lib/python3.12/site-packages/***/models/taskinstance.py", line 3185, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/***/.local/lib/python3.12/site-packages/***/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/***/.local/lib/python3.12/site-packages/***/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/***/.local/lib/python3.12/site-packages/***/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
  File "/home/***/.local/lib/python3.12/site-packages/***/providers/cncf/kubernetes/operators/pod.py", line 583, in execute
    return self.execute_sync(context)
  File "/home/***/.local/lib/python3.12/site-packages/***/providers/cncf/kubernetes/operators/pod.py", line 619, in execute_sync
    self.await_pod_completion(pod=self.pod)
  File "/home/***/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 336, in wrapped_f
    return copy(f, *args, **kw)
  File "/home/***/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 478, in __call__
    result = fn(*args, **kwargs)
  File "/home/***/.local/lib/python3.12/site-packages/***/providers/cncf/kubernetes/operators/pod.py", line 670, in await_pod_completion
    self.pod_manager.fetch_requested_container_logs(
  File "/home/***/.local/lib/python3.12/site-packages/***/providers/cncf/kubernetes/utils/pod_manager.py", line 618, in fetch_requested_container_logs
    status = self.fetch_container_logs(pod=pod, container_name=c, follow=follow_logs)
  File "/home/***/.local/lib/python3.12/site-packages/***/providers/cncf/kubernetes/utils/pod_manager.py", line 516, in fetch_container_logs
    last_log_time, exc = consume_logs(since_time=last_log_time)
  File "/home/***/.local/lib/python3.12/site-packages/***/providers/cncf/kubernetes/utils/pod_manager.py", line 459, in consume_logs
    for raw_line in logs:
  File "/home/***/.local/lib/python3.12/site-packages/***/providers/cncf/kubernetes/utils/pod_manager.py", line 263, in __iter__
    for data_chunk in self.response.stream(amt=None, decode_content=True):
  File "/home/***/.local/lib/python3.12/site-packages/urllib3/response.py", line 1063, in stream
    yield from self.read_chunked(amt, decode_content=decode_content)
  File "/home/***/.local/lib/python3.12/site-packages/urllib3/response.py", line 1219, in read_chunked
    self._update_chunk_length()
  File "/home/***/.local/lib/python3.12/site-packages/urllib3/response.py", line 1138, in _update_chunk_length
    line = self._fp.fp.readline()  # type: ignore[union-attr]
  File "/usr/local/lib/python3.12/socket.py", line 720, in readinto
    return self._sock.recv_into(b)
  File "/home/***/.local/lib/python3.12/site-packages/***/models/taskinstance.py", line 3095, in signal_handler
    self.log.error("Stacktrace: \n%s", "".join(traceback.format_stack()))

[2025-05-23T04:04:42.437+0000] {pod_manager.py:496} INFO - [base] downloading https://repo1.maven.org/maven2/org/apache/commons/commons-text/1.8/commons-text-1.8.jar ...
[2025-05-23T04:04:42.482+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-23T04:04:42.513+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=16326, status='terminated', exitcode=0, started='02:13:41') (16326) terminated with exit code 0
